XpringKit/XRPTransaction.swift

```diff
@@ -3,22 +3,56 @@ import Foundation
 /// A transaction on the XRP Ledger.
 ///
 /// - SeeAlso: https://xrpl.org/transaction-formats.html
-// TODO(keefertaylor): Modify this object to use X-Address format.
+/// TODO(keefertaylor): Modify this object to use X-Address format.
```

Comment:
Q: Are all comments in Swift `///`? Even TODOs?

Reply:
Good question - I learned that no they're not:  multiline comments with `/** **/` and single line comments with `///` will show up in the generated interface, whereas multis with `/* */` and singles with just `//` won't.  This has been reverted.